pontoon/sync/changeset.py

```diff
@@ -191,6 +197,13 @@ def execute_create_db(self):
                         fuzzy=vcs_translation.fuzzy
                     ))
 
+        Term.objects.assign_terms_to_entities([(
```

Comment:
This sounds like it's making changes to the ``Entity`` data. If so, why is that function in the ``Term`` model namespace?